admin/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# the goal of this script is to inject environment variables into the frontend
+# by creating a javascript file that will be injected in the boot file
+WWW_DIR=/app
+ENV_PATH="${WWW_DIR}/env.js"
+# Create the file
+echo -n "window.env = {" > "${ENV_PATH}"
+COMMA=""
+for envrow in $(printenv); do
+  IFS='=' read -r key value <<< "${envrow}"
+  case "${key}" in
+    "API_URL" | "API_PATH" | "AUTH_CLIENT_ID")
+      echo -n "${COMMA}" >> "${ENV_PATH}"
+      echo -n "${key}: '${value}'" >> "${ENV_PATH}"
+      COMMA=","
+      ;;
+    *)
+      # ignore the key
+      ;;
+  esac
+done
+echo "};" >> "${ENV_PATH}"
```

Comment:
if the env.js file already exist in your repo.. it will just add it ? because you're using >> instead of >


right ? @ymarcon

Reply:
At line 8 it overrides it first.

Reply:
J'avais pas vu :D 👍